common/src/main/java/tachyon/util/network/NetworkAddressUtils.java

```diff
@@ -44,6 +45,7 @@
 /**
  * Common network address related utilities shared by all components in Tachyon.
  */
+@ThreadSafe
```

Comment:
is it necessary to make a util class thread-safe? It seems as long as we move the local ip/hostname resolution to static init, this class becomes read-only, isn't it?